### PR TITLE
Gokhanap community members2

### LINKED ISSUE #153: [full] community members
we don't have the designs yet. just create a simple page, and render all the members on a new page. 

route:
/community/${communityUuid}/members

Render following info for each member:

Firstname - Lastname
Role (in the community, admin, member etc..)

Next to each member, render a pencil icon (edit icon). admins should be able to remove members from the community.

Put `invite members` button at the top of the page.

### PR DESCRIPTION
**Pull request by gokhanap**
### Fixes #153 

route:
- [x] /community/${communityUuid}/members

#### Render following info for each member:
- [x] Firstname - Lastname
- [x] Role (in the community, admin, member etc..)
- [x] Next to each member, render a pencil icon (edit icon).
- [ ] admins should be able to remove members from the community.
- [x] Put invite members button at the top of the page

### CODE REVIEW

File: src/components/pages/community-members.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Title, Icon, Paragraph, Button } from '@/components/ui';
+
+class CommunityMembers extends React.Component {
+  constructor(props) {
```
Comment (trkaplan) [inline src/components/pages/community-members.js]: You can omit constructor. Babel will transpile your code and add a constructor for you behind the scenes. see: https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599

**Comment by SelmanKahya** [comment]
any updates?

**Comment by SelmanKahya** [comment]
merging, I am going to address remaining issues

**Comment by SelmanKahya** [comment]
see https://github.com/Kommunity-app/kommunity-frontend/pull/173 for the changes i made to this PR

**Comment by SelmanKahya** [comment]
please create a new PR for the missing features you meant to implement

**Pull request merged**